jest-lint.config.js

```diff
@@ -4,4 +4,5 @@ module.exports = Object.assign({}, baseConfig, {
     runner: 'jest-runner-eslint',
     displayName: 'lint',
     testMatch: ['<rootDir>/src/**/*.(ts|js)'],
+    watchPlugins: ['jest-runner-eslint/watch-fix'],
```

Comment:
this is nice!

Reply:
Yeah! It was painful not having that. Very nice to have it as a watch option.